packages/node/package.json

```diff
@@ -32,7 +32,7 @@
     "edge-runtime": "2.5.7",
     "esbuild": "0.14.47",
     "etag": "1.8.1",
-    "exit-hook": "2.2.1",
+    "exit-hook": "4.0.0",
```

Comment:
I'm not sure we want to start introducing ESM dependencies quite yet. The CLI is still being transpiled to CommonJS and while `esbuild` does help with ESM compatibility, it hasn't been fully tested and there are known issues with dependencies that ESM features like `import.meta`.

Does `exit-hook@2.2.1` not work?

Reply:
I think this should be evaluated on a case by case basis. Seems like the tests are passing, so I'm fine with this update, especially if it's necessary to get the relevant hook for this change.